sdrf-proteomics/README.adoc

```diff
@@ -190,7 +191,7 @@ NOTE: Additional characteristics can be added depending on the type of the exper
 
 Some important notes:
 
-- Each characteristic name in the column header SHOULD be a CV term from the EFO ontology. For example, the header _characteristics[organism]_ corresponds to the ontology term Organism.
+- Each characteristic name in the column header SHOULD be a CV term from the EFO ontology. For example, the header _characteristics[organism]_ corresponds to the ontology term Organism. However the values could be from EFO or other ontologies. For example, for diseases we RECOMMEND to use MONDO for diseases because it has better coverage than EFO.
```

Comment:
```suggestion
- Each characteristic name in the column header SHOULD be a CV term from the EFO ontology. For example, the header _characteristics[organism]_ corresponds to the ontology term Organism. However the values could be from EFO or other ontologies. For example, we RECOMMEND to use MONDO for diseases because it has better coverage than EFO.
```